src/core/ContentWindow.cpp

```diff
@@ -166,6 +169,38 @@ bool ContentWindow::isHidden() const
     return windowState_ == HIDDEN;
 }
 
+void ContentWindow::setSelected( bool value )
```

Comment:
Very verbose. ContentWindow::setState( enum value ) is not better? Or at least setState( value ? SELECTED : NONE );